results/customer-portal/installer-platform.detailed-description.html

```diff
@@ -0,0 +1,3 @@
+The Red Hat JBoss Developer Studio Platform Installer includes everything you need to start working with the Red Hat Container Development Kit and Red Hat JBoss Developer Studio, including OpenJDK 8, Cygwin, PuTTY, and OpenShift Client (OC) Tools. 
+
+It requires Vagrant 1.7.4 and Virtual Box 5.0.8 or later, but will install them for you if not already present on your 64-bit Windows system if a network is available.
```

Comment:
But those are the actual minimum versions. We don't support anything else.... Not even older/newer vagrant. Correct?